### PR TITLE
Buffs capacity of cigars to be greater than cigarettes, removes fucki… [WHOOPS]

### PR DESCRIPTION
**Pull request by Dorsisdwarf**
…ng stupid messages from nicotine.

Previously, cigars had a smaller capacity for chems than normal cigarettes, but had the same amount of nicotine in them.

Also this is technically a security buff, as people are more likely to play detective now without stupid messages.

### CODE REVIEW

**Comment by Thunder12345** [comment]
I just noticed, this picks a smoking message every time on_mob_life for nicotine is called, but only rarely uses the picked message. Would it be better to put the pick inside the prob, so it's not constantly running unnessecary picks?

File: code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm [hunk 1]
```diff
@@ -39,8 +39,8 @@
 	addiction_threshold = 30
 
 /datum/reagent/drug/nicotine/on_mob_life(var/mob/living/M as mob)
-	var/smoke_message = pick("You can just feel your lungs dying!", "You feel relaxed.", "You feel calmed.", "You feel the lung cancer forming.", "You feel the money you wasted.", "You feel like a space cowboy.", "You feel rugged.")
-	if(prob(5))
+	var/smoke_message = pick("You feel relaxed.", "You feel calmed.","You feel the need to slap Goofball six kinds of shit.")
```
Comment (Dorsisdwarf) [inline code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm]: Guess who forgot to remove a joke message -.-

File: code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm [hunk 2]
```diff
@@ -39,8 +39,8 @@
 	addiction_threshold = 30
 
 /datum/reagent/drug/nicotine/on_mob_life(var/mob/living/M as mob)
-	var/smoke_message = pick("You can just feel your lungs dying!", "You feel relaxed.", "You feel calmed.", "You feel the lung cancer forming.", "You feel the money you wasted.", "You feel like a space cowboy.", "You feel rugged.")
-	if(prob(5))
+	if(prob(1))
+		var/smoke_message = pick("You feel relaxed.", "You feel calmed.","You feel alert")
```
Comment (PKPenguin321) [inline code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm]: "You feel alert" needs a period at the end to be consistent with the other messages

Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm]: And re-add the space cowboy and rugged ones, those were a nod to malboros

**Comment by Iamgoofball** [comment]
:+1: after you re-add 2 of the messages.

**Comment by PKPenguin321** [comment]
Yeah, to be honest the cowboy and rugged ones have nothing wrong with them so why bother removing them

**Comment by Dorsisdwarf** [comment]
I'll compromise by adding in "rugged".

**Pull request merged**